src/main/kotlin/Main.kt

```diff
@@ -8,4 +8,16 @@ fun main() {
   // Create account, generate keypairs
   //  val newAccountKeypair = wallet.create()
   //  println(newAccountKeypair)
+
```

Comment:
If we do not intend to run the SDK, we may remove the Main.kt. 

If we want something that can be run to test developing code, JUnit is our friend.